### PR TITLE
Rename from Machine Deletion to Machine Deletion Remediation

### PR DESCRIPTION
**Pull request by clobrano**

### CODE REVIEW

File: controllers/machinedeletionremediation_controller.go [hunk 1]
```diff
@@ -27,7 +29,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"strings"
 
 	"github.com/medik8s/machine-deletion/api/v1alpha1"
```
Comment (razo7) [inline controllers/machinedeletionremediation_controller.go:33]: Do we plan to change the repo name after the PR?
`github.com/medik8s/machine-deletion` -> `github.com/medik8s/machine-deletion-remediation`

Comment (clobrano) [reply controllers/machinedeletionremediation_controller.go:33]: yes, absolutely. I wanted something working with the current path, then change it

**Comment by razo7** [comment]
Can you add [E2E test on OCP CI](https://github.com/openshift/release/blob/master/ci-operator/config/medik8s/machine-deletion/medik8s-machine-deletion-main.yaml#L26) since MDR already has E2E tests (https://github.com/medik8s/machine-deletion/pull/32)?

**Comment by clobrano** [comment]
> Can you please add [E2E test on OCP CI](https://github.com/openshift/release/blob/master/ci-operator/config/medik8s/machine-deletion/medik8s-machine-deletion-main.yaml#L26) since MDR already has E2E tests (#32)?

sure

**Comment by mshitrit** [comment]
I think you might have missed bundle docker file [here](https://github.com/medik8s/machine-deletion/blob/main/bundle.Dockerfile#L7)

**Comment by clobrano** [comment]
> I think you might have missed bundle docker file [here](https://github.com/medik8s/machine-deletion/blob/main/bundle.Dockerfile#L7)

indeed, thank you

**Review by slintes — CHANGES_REQUESTED** [review]
Can you change the module name as well please (go.mod and imports)?

nits only, up to you if you want to change names there (and IMHO for variable names a short `mdr` would be fine):
- https://github.com/medik8s/machine-deletion/pull/33/files#diff-e91d29c6d4c58503d9c2d72e2b04f1b70901a613e23008d1d1cfe8beff7038d4R25
- https://github.com/medik8s/machine-deletion/pull/33/files#diff-867a8834813cdf408e2387961fc75a31146379802d5abec6f465f1d8a565b903R273

**Comment by clobrano** [comment]
> Can you change the module name as well please (go.mod and imports)?

regarding this, it would be better to change the repo URL first, or module name and imports?
Or, it doesn't make sense, since there would still be a moment of misalignment? :thinking:

**Comment by clobrano** [comment]
I just noticed that changing the module path in `go.mod` stops the LSP server complaining about import paths as well

**Review by slintes — APPROVED** [review]

**Pull request merged**